### PR TITLE
Create bluebeetle.ms

### PR DESCRIPTION
**Pull request by MrCraft888**

### CODE REVIEW

File: CommandHelper/LocalPackages/commands/buffs/bluebeetle.ms [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            'op': 'false',
+            'speech': "Another annoying monster battle, another interrupted date... another typical night in the life of Jaime Reyes",            
+            'powers': array(
+                "/buff",
+                "/carryplayer",
+                "/dismountplayer",
+                "/jump 0-2",
+                "/run 0-3",
+                "/haste 0-5",
+                "/flyspeed <number>",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/bluebeetle.ms]: requires proof

File: CommandHelper/LocalPackages/commands/buffs/bluebeetle.ms [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+                "/fly"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 6 true",
+                "/effect ". @plr ." 11 9999 2 true",
+                "/effect ". @plr ." 21 9999 9 true",
+                "/shot give ". @plr ." quantumlaser"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/bluebeetle.ms]: requires proof

Comment (MrCraft888) [inline CommandHelper/LocalPackages/commands/buffs/bluebeetle.ms]: http://s346.photobucket.com/user/destroyah87/media/BlueBeetle.jpg.html

File: CommandHelper/LocalPackages/commands/buffs/bluebeetle.ms (hunk 1 above)
Comment (MrCraft888) [inline CommandHelper/LocalPackages/commands/buffs/bluebeetle.ms]: https://s-media-cache-ak0.pinimg.com/236x/df/a4/c7/dfa4c72587ac6552e50741d59fc0aa86.jpg

Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/bluebeetle.ms]: thats just regular /fly, nothing super impressive.

Please find another pic or remove flyspeed.

File: CommandHelper/LocalPackages/commands/buffs/bluebeetle.ms (hunk 2 above)
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/bluebeetle.ms]: you should make his own weapon for that

Comment (MrCraft888) [inline CommandHelper/LocalPackages/commands/buffs/bluebeetle.ms]: im fine

Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/bluebeetle.ms]: it's not really a choice, quantum laser is for iron man

**Pull request closed without merging**